tokio/src/lib.rs

```diff
@@ -145,11 +142,20 @@ if_runtime! {
 
 // ===== Experimental async/await support =====
 
+#[cfg(feature = "async-await-preview")]
+extern crate tokio_futures;
+
+#[cfg(feature = "async-await-preview")]
+extern crate tokio_macros;
+
 #[cfg(feature = "async-await-preview")]
 mod async_await;
 
 #[cfg(feature = "async-await-preview")]
 pub use async_await::{run_async, spawn_async};
 
 #[cfg(feature = "async-await-preview")]
-pub use tokio_async_await::await;
+pub use tokio_futures::await;
+
+#[cfg(feature = "async-await-preview")]
+pub use tokio_macros::{main, test};
```

Comment:
can we get a version that isnt async fn?

Reply:
We could consider adding one... but assuming that all this stuff is arriving on stable soon, is it worth it?

Reply:
can we do the same as we do with the _async fns?

`#[tokio::test_async]`

then for 0.1 we can use `#[tokio::test]`? then in 0.2 breaking changes we switch it back for 0.3.

Reply:
I'm open to switching this up in a follow up PR.

However, it is less obvious how it would work with 0.1 because things like `await` is not easily provided, yet useful in tests.

Reply:
It would also be possible to use the same macro for 0.3 and 0.1. The macro can look for the `async` keyword.